hawc/apps/riskofbias/constants.py

```diff
@@ -54,6 +54,8 @@ class RiskOfBiasResponses(models.IntegerChoices):
 
 NA_SCORES: Tuple[int, ...] = (10, 20)
 
+DEFAULT_SCORES: Tuple[int, ...] = (0, 12, 22)
```

Comment:
remove; use this instead https://github.com/shapiromatron/hawc/blob/next/hawc/apps/riskofbias/constants.py#L22-L28